packages/nc-gui/components/cell/ClampedText.vue

```diff
@@ -26,7 +26,7 @@ onMounted(() => {
     <text-clamp
       :key="`clamp-${key}-${props.value?.toString().length || 0}`"
       class="w-full h-full break-all"
-      :text="`${props.value || ''}`"
+      :text="`${props.value || ' '}`"
```

Comment:
Any reason to provide whitespace?

Reply:
The component itself wont trigger update for `''` so it' ll keep the previous cell result.

[screen-capture (13) (1).webm](https://user-images.githubusercontent.com/35857179/219010580-19af3fa4-0853-4525-b8ba-2719bfd842cc.webm)